config/locales/en.yml

```diff
@@ -29,3 +29,4 @@ en:
       long: "%e %B %Y %H:%M"
       long_with_seconds: "%e %B %Y %H:%M:%S"
       time: "%H:%M"
+      official: "%l:%M%P on %A %-d %B %Y"
```

Comment:
I'm not entirely sure if it's a format we need to be using across the app. I'll follow up with design on this next week.

The new format comes from the guidance here: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#times

It _felt_ "official" 😆 since it's based on the official guidance. But yes, it could end up replacing `long`.

Reply:
@Nitemaeric in the interest of descriptive names, could you please rename this to `time_on_date`? I think this is more descriptive than "official". 🙏🏻

Reply:
Updated! :D